### PR TITLE
(582) Required first name and last name

### PR DESCRIPTION
**Pull request by jkempster34**
### Update the `users:create` Task to accept name attributes
Users now have a `first_name` and `last_name`. 

This updates the `users:create` Task, so that they can be passed as Task arguments, and updates the relevant documentation.

### Validate presence of name attributes on User
Currently, the `first_name` and `last_name` properties of the User model are optional.

This adds validation to check that all users do have `first_name` and `last_name`. 

There is no need for any data migrations as we don't yet have a deployed application.

### Remove fallbacks for when the name attributes of User are absent
We have various bit of logic dotted around to handle users that don’t have `first_name` and `last_name`.

Now that these are required, we can safely remove any fallbacks.

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by jkempster34** [comment]
[Require Users to have `first_name` and `last_name`](https://trello.com/c/zIWkb4mR/582-require-users-to-have-firstname-and-lastname)

File: spec/requests/sign_in_spec.rb [hunk 1]
```diff
@@ -27,10 +27,7 @@
   end
 
   context "when the users is not known by the application" do
-    before do
-      User.create!(email: "another.user@education.gov.uk")
```
Comment (jacksonj04) [inline spec/requests/sign_in_spec.rb:31]: This is a change in the behaviour of the test; what was the significance (or given the fact that deleting it still passes, what should the significance have been) of `another.user@education.gov.uk`?

**Review by jacksonj04 — APPROVED** [review]
I love PRs which delete more than they add.

One question about a changing test behaviour, but other than that this looks good and wholesome to me.

File: spec/requests/sign_in_spec.rb (hunk 1 above)
Comment (jkempster34) [reply spec/requests/sign_in_spec.rb:31]: Ah, yeah, I forgot about that. That line doesn't seem to do anything. 

I'll double check what the test is trying to assert and rewrite if necessary 👀

**Pull request merged**